cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -839,14 +854,16 @@ public void workflowSchedules(HttpRequest request, HttpResponder responder,
    * Get schedule state.
    */
   @GET
-  @Path("/apps/{app-id}/workflows/{workflow-id}/schedules/{schedule-id}/status")
+  @Path("/apps/{app-id}/workflows/{workflow-id}/schedules/{schedule-name}/status")
```

Comment:
I don't know what a "runnable" is. Is that a program? 
Because the application can contain schedules, the schedules endpoint should be nested under the app. Nesting it under the workflow suggests that the workflow contains schedules, and I thought we just decided that schedules are not part of the workflow.